android/app/src/main/AndroidManifest.xml

```diff
@@ -37,39 +42,56 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+
+            <meta-data
+                android:name="android.app.default_searchable"
+                android:value=".activities.SearchActivity" />
         </activity>
         <activity
             android:name=".activities.SettingsActivity"
-            android:label="@string/action_settings"
+            android:configChanges="orientation|screenSize|keyboardHidden"
             android:exported="false"
-            android:theme="@style/AppTheme.NoActionBar.Alternate"
-            android:configChanges="orientation|screenSize|keyboardHidden">
+            android:label="@string/action_settings"
+            android:theme="@style/AppTheme.NoActionBar.Alternate">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+
                 <category android:name="android.intent.category.NOTIFICATION_PREFERENCES" />
             </intent-filter>
         </activity>
         <activity
             android:name=".activities.TrackSessionsActivity"
+            android:configChanges="orientation|screenSize|keyboardHidden"
             android:label="@string/title_section_tracks"
-            android:theme="@style/AppTheme.NoActionBar.Alternate"
-            android:configChanges="orientation|screenSize|keyboardHidden"/>
+            android:theme="@style/AppTheme.NoActionBar.Alternate" />
         <activity
             android:name=".activities.LocationActivity"
+            android:configChanges="orientation|screenSize|keyboardHidden"
             android:label="@string/title_section_location"
-            android:theme="@style/AppTheme.NoActionBar.Alternate"
-            android:configChanges="orientation|screenSize|keyboardHidden"/>
+            android:theme="@style/AppTheme.NoActionBar.Alternate" />
         <activity
             android:name=".activities.SpeakerDetailsActivity"
+            android:configChanges="orientation|screenSize|keyboardHidden"
             android:label="@string/title_section_speakerlist"
-            android:theme="@style/AppTheme.NoActionBar"
-            android:configChanges="orientation|screenSize|keyboardHidden"/>
-
+            android:theme="@style/AppTheme.NoActionBar" />
         <activity
             android:name=".activities.SessionDetailActivity"
+            android:configChanges="orientation|screenSize|keyboardHidden"
             android:label="@string/session"
-            android:theme="@style/AppTheme.NoActionBar"
-            android:configChanges="orientation|screenSize|keyboardHidden">
+            android:theme="@style/AppTheme.NoActionBar">
+            <meta-data
+                android:name="android.app.searchable"
+                android:resource="@xml/searchable" />
+        </activity>
+
+        <activity
+                android:name=".activities.SearchActivity"
```

Comment:
Similarly here, this block of code should have been the only one to be appearing in the diff